hazelcast/src/main/java/com/hazelcast/sql/impl/SqlDataSerializerHook.java

```diff
@@ -246,6 +249,7 @@ public DataSerializableFactory createFactory() {
         constructors[EXPRESSION_SIGN] = arg -> new SignFunction<>();
         constructors[EXPRESSION_RAND] = arg -> new RandFunction();
         constructors[EXPRESSION_DOUBLE] = arg -> new DoubleFunction();
+        constructors[EXPRESSION_DOUBLE_DOUBLE] = arg -> new DoubleBiFunction();
```

Comment:
Here, the constructors seem to be already ordered by `classId` rather than name.

Reply:
Done.